concepticondata/conceptlists.tsv

```diff
@@ -353,3 +353,4 @@ Zalizniak-2020-2591	Zalizniak, Anna A. and Smirnitskaya, Anna and Russo, Maksim
 LSDO-2019-462	Language and Social Development Organization	2019		462	basic, areal	English	Kuki-chin languages		LSDO2019		This list was used as a questionnaire for an investigation of Chin dialects, carried out by the Language and Social Development Organization in 2019. Nathan Statezni was so kind to share the original data with us.		
 List-2020-1365	List, Johann-Mattis	2020		1365	annotated	English	German	https://calc.hypotheses.org/2545	List2020		This concept list was originally compiled to provide German elicitation glosses for the [Intercontinental Dictionary Series](:ref:Key-2016-1310), but since some difficulties in the exact concepts were detected, the list was modified and now offers several elicitation glosses for one and the same elicitation item in the Intercontinental Dictionary Series, where this was assumed to be important by the author.		
 Starostin-2017-401	Starostin, George and Kassian, Alexei and Trofimov, Artem and Zhivlov, Mikhail	2017		401	basic, areal	Russian, English	Nostratic languages	https://starling.rinet.ru/new400/	Starostin2017		This concept list, provided in the form of a database, was compiled to perform lexicostatistical analyses on certain Eurasian languages and proto-languages which might have a Nostratic ancestor. The words were chosen to represent basic meanings common to most languages. Furthermore, semantic links were provided which link several closely connected words within the list.		
+Cowen-2020-16	Cowen, Alan S. and Keltner, Dacher and Schroff, Florian and Jou, Brendan and Adam, Hartwig and Prasad, Gautam	2020		16	specific	English	Arabic, English, French, German, Indonesian, Italian, Japanese, Korean, Mandarin, Polish, Portuguese, Russian, Spanish, Thai, Turkish, Vietnamese	https://doi.org/10.1038/s41586-020-3037-7	Cowen2020		This list of 16 emotions emerged as a result of a machine-learning analysis of large-scaled video material from 144 countries. The purpose of the study was to find out what facial expressions are associated with various naturalistic social contexts and determine culturally universal patterns of context–expression association.		
```

Comment:
If all these languages are mentioned as "target languages", we need them also in our concept list. If the data cannot be found in the supplement anywhere, the additional languages should be discarded.

Reply:
As far as I understood, the list itself is made only in English while the material analyzed was derived from the videos in the languages listed. But I can see now that they should be removed from here. Shall I then just indicate 'global' in the 'target languages'?

Reply:
And I'll add the rest of the concept sets missing here.